docs/cita/getting-started/setup.md

```diff
@@ -72,7 +71,7 @@ title: 环境准备
 2. 下载 CITA 安装包
 
    ```shell
-   $ wget https://github.com/cryptape/cita/releases/download/v0.24.0/cita_secp256k1_sha3.tar.gz
+   $ wget https://github.com/cryptape/cita/releases/download/v0.25.0/cita_secp256k1_sha3.tar.gz
```

Comment:
You can use this: `https://github.com/cryptape/cita/releases/latest` to avoid to fix it every time.